gui/wxpython/datacatalog/tree.py

```diff
@@ -2261,7 +2261,7 @@ def _popupMenuElement(self):
     def _popupMenuMultipleLocations(self):
         """Create popup menu for multiple selected locations"""
         menu = Menu()
-        item = wx.MenuItem(menu, wx.ID_ANY, _("&Delete locations"))
+        item = wx.MenuItem(menu, wx.ID_ANY, _("&Delete projects"))
```

Comment:
```suggestion
        item = wx.MenuItem(menu, wx.ID_ANY, _("&Delete projects (location)"))
```

(to keep coherence with line 2222)